libqsopt/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+	<name>libqsopt</name>
+	<version>0.6.10</version>
+	<description>Wrapper for the Qsopt linear programming solver</description>
+
+	<url type="website">http://www.math.uwaterloo.ca/~bico/qsopt/index.html</url>
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+
+	<license>GPL</license>
```

Comment:
Note though, that GPL is bad. This is a strong copyleft license which forces all programs linking against this library  to be licensed under the GPL as well.

Furthermore, GPL cannot be distributed without the source code. So as long as you don't have this one, this is, in general not a GPL license.